lib/Brass/API.pm

```diff
@@ -98,16 +99,12 @@ get 'api/pwd/' => sub {
     my $action = query_parameters->get('action')
         or error __"Need required action";
 
-    $action eq 'sqldb' || $action eq 'admonitor' || $action eq 'system' || $action eq 'wazuh'
+    Brass::Actions::is_allowed_action($action)
         or error __x"Invalid action: {action}", action => $action;
 
     my $param = query_parameters->get('param');
-    $action eq 'sqldb' && !$param
-        and error __"Please specify required username for SQL password";
-    $action eq 'system' && !$param
-        and error __"Please specify required username for system password";
-    $action eq 'wazuh' && !$param
-        and error __"Please specify required username for wazuh password";
+    !$param
+        and error __"Please specify required username for $action password";
```

Comment:
As a general point, this is better written as ```__x"Please specify required username for {action} password", action => $action```. This abstracts potential future translations, not that that will even happen!

Reply:
I'll change that to make it consistent with the rest of the code.